### PR TITLE
sexbabevr patch

### PR DESCRIPTION
**Pull request by smvemjsuneptune**
site updated
update page url
update xpath selectors

### CODE REVIEW

File: Contents/Code/PAutils.py [hunk 1]
```diff
@@ -389,7 +389,7 @@ def postParseTitle(output):
     replace = [('“', '\"'), ('”', '\"'), ('’', '\''), ('W/', 'w/'), ('Aj', 'AJ')]
 
     # Add space after a punctuation if missing
-    output = re.sub(r'(?=[\!|\:|\?|\.](?=(\w{2,}))\b)\S(?!(co\b|net\b|com\b|org\b|porn\b|E\d|xxx\b))', lambda m: m.group(0) + ' ', output, flags=re.IGNORECASE)
+    output = re.sub(r'(?=[\!|\:|\?|\.](?=(\w{2,}))\b)\S(?!(co\b|net\b|com\b|org\b|porn\b|E\d|\b))', lambda m: m.group(0) + ' ', output, flags=re.IGNORECASE)
```
Comment (smvemjsuneptune) [inline Contents/Code/PAutils.py:392]: needed to match exact id e.g., `https://realjamvr.com/scene/kill-bill-o-ren-ishii-a-xxx-parody/`

File: Contents/Code/PAutils.py [hunk 2]
```diff
@@ -458,7 +458,7 @@ def cleanHTML(text):
 def getCleanSearchTitle(title):
     trashTitle = (
         'RARBG', 'COM', r'\d{3,4}x\d{3,4}', 'HEVC', r'H\d{3}', 'AVC', r'\dK',
-        r'\d{3,4}p', 'TOWN.AG_', 'XXX', 'MP4', 'KLEENEX', 'SD', 'HD',
+        r'\d{3,4}p', 'TOWN.AG_', 'MP4', 'KLEENEX', 'SD', 'HD',
```
Comment (smvemjsuneptune) [inline Contents/Code/PAutils.py:461]: needed to match exact id e.g., https://realjamvr.com/scene/kill-bill-o-ren-ishii-a-xxx-parody/

File: Contents/Code/networkHighTechVR.py [hunk 3]
```diff
@@ -72,11 +78,11 @@ def update(metadata, lang, siteNum, movieGenres, movieActors, art):
...
-        tagline = rawtagline.split('|')[1].strip()
-    elif '-' in rawtagline:
-        tagline = rawtagline.split('-')[0].strip()
+    tagline = detailsPageElements.xpath('//title')[0].text_content().strip()
+    if '|' in tagline:
+        tagline = tagline.split('|')[1].strip()
+    elif '-' in tagline:
+        tagline = tagline.split('-')[0].strip()
 
     metadata.tagline = tagline
```
Comment (smvemjsuneptune) [inline Contents/Code/networkHighTechVR.py:87]: if neither conditional hits tagline is undefined

File: postprocessing2/PAsearchSites.py [hunk 4]
```diff
@@ -1059,7 +1057,6 @@ def getSearchSiteIDByFilter(searchFilter):
             siteNameF = sites[0].lower().replace(" ", "").replace("'", "")
 
             if siteNameF in searchFilterF[0] or siteNameF in searchFilterF[1]:
-                ##print('Site found with method #1')
```
Comment (smvemjsuneptune) [inline postprocessing2/PAsearchSites.py:1062]: linter was complaining about these, so removed

File: Contents/Code/PAutils.py (hunk 2 above)
Comment (DirtyRacer1337) [reply Contents/Code/PAutils.py:461]: use then `searchData.filename` not `searchData.title`, it can break other sites

Comment (smvemjsuneptune) [reply Contents/Code/PAutils.py]: reverted

**Pull request merged**